### ISSUE TITLE
Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift

### ISSUE THREAD

**Issue by oba18** [oba18/survey-paper#29]
# 論文まとめ

## 0. 論文
[Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167)
![image](https://user-images.githubusercontent.com/33748835/79191069-b8c37980-7e60-11ea-9b5a-5b91ba39e7c4.png)

## 1. どんなもの？
Batch Normalizationという正規化をすることで，高い学習率を保つことができる．

## 2. 先行研究と比べてどこがすごい？
ディープラーニングの学習で，前の層のパラメータが変化すると各層の入力の分布がへんかするという事実によって複雑になる．学習率が低下したり，慎重なパラメータの初期化が必要になり，学習が遅くなる．それに対して，正則化モデルをあー来てlクチャの一部とし，学習ミニバッチごとに正則化を行うことでそれを改善した．

## 3. 技術や手法のキモはどこ？
Batch Normalization という正規化の方法．
活性化を正規化し，この正規化をネットワークアーキテクチャ事態に組み込むことで正規化が適切に処理される．ミニバッチごとに正規化を実行し，正規化パラメータを介して勾配をバックプロパゲーションする．
![image](https://user-images.githubusercontent.com/33748835/79192109-0f31b780-7e63-11ea-9ca4-226ddb1860a4.png)
Dropoutを必要としていない．

## 4. どうやって有効だと検証した？
ImageNetデータセットを用いて検証した．
